src/lightning/pytorch/callbacks/progress/rich_progress.py

```diff
@@ -362,7 +362,7 @@ def on_sanity_check_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningMod
     def on_train_epoch_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
         if self.is_disabled:
             return
-        total_batches = self.total_batches_current_epoch
+        total_batches = self.total_train_batches
         train_description = self._get_train_description(trainer.current_epoch)
 
         if self.main_progress_bar_id is not None and self._leave:
```

Comment:
Maybe we should rename it to `train_progress_bar` now for both files. Maybe in a follow-up

Reply:
Yes. However, it could also be that there are few users who made a custom bar relying on this public attribute.